distribution/kernel/carbon-home/repository/resources/conf/default.json

```diff
@@ -153,5 +153,22 @@
   "truststore.password": "wso2carbon",
   "encryption.type": "asymmetric",
   "encryption.algorithm": "AES",
-  "encryption.alias": "symmetric.key.value"
-}
\ No newline at end of file
+  "encryption.alias": "symmetric.key.value",
+
+  "authenticator.token_ui_authenticator.disable": true,
```

Comment:
make these "*.enabled" : false instead of disabled true, which will result in double negations